ledger/participant-state/src/test/suite/scala/com/daml/ledger/participant/state/TimeModelTest.scala

```diff
@@ -89,6 +89,24 @@ class TimeModelTest extends WordSpec with Matchers {
           .checkTime(referenceTime.minus(instance.minSkew).minus(epsilon), referenceTime)
           .isLeft shouldEqual true
       }
+
+      "produce a valid error message" in {
+        val timeModel =
+          TimeModel(
+            avgTransactionLatency = Duration.ZERO,
+            minSkew = Duration.ofSeconds(10L),
+            maxSkew = Duration.ofSeconds(20L),
+          ).get
+        val ledgerTime = "2000-01-01T12:00:00Z"
+        val recordTime = "2000-01-01T12:30:00Z"
+        val lowerBound = "2000-01-01T12:29:50Z"
+        val upperBound = "2000-01-01T12:30:20Z"
+        val expectedMessage = s"Record time $recordTime outside of range [$lowerBound, $upperBound]"
```

Comment:
But… it's not. The ledger time is outside the range. The record time is within it.